docker/ranger/Dockerfile_slim

```diff
@@ -1,24 +1,24 @@
 FROM ubuntu:18.04
-MAINTAINER Grisha Hushchyn <gh13@sanger.ac.uk>
+LABEL org.opencontainers.image.authors="Grisha Hushchyn <gh13@sanger.ac.uk>"
 
 # Hold versions in environment variables.
 ENV NODE_VERSION=10.16.3 \
 #   MONGODB_VERSION=3.6.14 \
     SAMTOOLS_VERSION=1.10+42_g3c4b380 \
```

Comment:
```suggestion
    SAMTOOLS_VERSION=1.14.0 \
```

Reply:
Same here for `irods-runtime=4.2.7`

Reply:
Bumped the samtools version in the three dockerfiles, but only added `irod-runtime=4.2.7` in two of them. It didn't seem that it was needed in `docker/ranger/Dockerfile` but maybe I'm wrong?